worker-workflow/src/main/java/com/github/cafdataprocessing/WorkflowEvaluator.java

```diff
@@ -53,10 +50,11 @@ public static void evaluate(final Document document, final String workflowAsJava
      * @param document Document to set post processing information on.
      * @param workflowStorageReference DataStore reference for a workflow that can be executed in post processing.
      */
-    private static void setWorkflowPostProcessingScript(final Document document, final String workflowStorageReference)
+    private static void setWorkflowPostProcessingScript(final Document document, final String workflowStorageReference) throws ScriptException
```

Comment:
It separated into a separate method to make the code clearer. I prefer for the 'evaluate' method to have a single line that can be read as 'set the workflow script' instead of the actual implementation of setting the script which is a more verbose. If the logic to set the script needs to be updated we're doing it under an easily identifiable method also rather than part of a larger one.